packages/engine-core/package.json

```diff
@@ -59,7 +59,7 @@
     "new-github-issue-url": "0.2.1",
     "p-retry": "4.6.1",
     "terminal-link": "2.1.1",
-    "undici": "3.3.6"
+    "undici": "4.10.0"
```

Comment:
Undici is at v4.12.1 now - do we want to let these updates happen via Renovate later?

Reply:
Better to do it know indeed.